Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -292,6 +304,84 @@ inline bool _ShouldDrawToCommandList() {
     HRESULT _CreateShadowEffect(ID2D1Image* inputImage, ID2D1Effect** outShadowEffect);
 
 public:
+    inline D2D1_ANTIALIAS_MODE GetAntialiasMode() {
+        if (CurrentGState().shouldAntialias == _kCGToggleDefault && allowsAntialiasing == _kCGToggleDefault) {
+            return D2D1_ANTIALIAS_MODE_PER_PRIMITIVE;
+        }
+
+        return CurrentGState().shouldAntialias == _kCGToggleOn && allowsAntialiasing == _kCGToggleOn ? D2D1_ANTIALIAS_MODE_PER_PRIMITIVE :
+                                                                                                       D2D1_ANTIALIAS_MODE_ALIASED;
+    }
+
+    inline D2D1_TEXT_ANTIALIAS_MODE GetTextAntialiasMode() {
+        if (CurrentGState().shouldSmoothFonts == _kCGToggleDefault && allowsFontSmoothing == _kCGToggleDefault) {
+            return D2D1_TEXT_ANTIALIAS_MODE_DEFAULT;
```

Comment:
I think I misunderstood what you meant when you said Default. I assumed that _nothing would be set on the render target_ if the value was "default"